model-optimizer/mo/utils/versions_checker.py

```diff
@@ -127,6 +145,9 @@ def get_module_version_list_from_file(file_name, env_setup):
     req_dict = list()
     with open(file_name) as f:
         for line in f:
+            # Handle comments
```

Comment:
for consistency in comments please start a comment with lower-case letter

Reply:
Done.